utils/wasm-builder/src/wasm_project.rs

```diff
@@ -91,9 +91,12 @@ impl WasmProject {
             .to_string_lossy()
             .into();
 
+        let target_dir_name = env::var("CARGO_TARGET_DIR")
```

Comment:
I'd split it into different pr but ok

Reply:
@StackOverflowExcept1on ptal

Reply:
lgtm, this code is trying to not hardcode the target directory